### DISCUSSION TITLE
Getting "You are not logged into any accounts on pers"

### DISCUSSION THREAD

**Discussion by phanorcoll** [pwntester/octo.nvim#505] [Q&A] [answered]
I have gh configured fine, I'm able to run the command from the terminal and as a command from Neovim, but when trying to run Octo, I get the error "You are not logged into any accounts on pers".

am I missing something, maybe a configuration step.?

**Comment by benniel** [comment]
@phanorcoll were you able to resolve this? I'm having the same issue. `gh pr list` works fine from the command line.

**Reply by benniel** [reply, marked as answer]
After some digging I managed to find the issue:

1. I'm using ssh aliases in `~/.ssh/config` to link private keys to specific hosts.
2. I have an alias named `personal_github`, which points to `github.com`
3. `git remote` in this repository is configured as `git@personal_github:<username>/<repo>` 
4. Octo grabs the `host_url` from the output of `git remote -v`, which returns `personal_github`
5. Then Octo runs `gh auth status --hostname personal_github`, which returns the message `You are not logged into any accounts on personal_github`.

The solution is to add the ssh alias to Octo's setup:
```lua
    require("octo").setup({
        ssh_aliases = { personal_github = "github.com" },
    })
```

**Reply by PeterCardenas** [reply]
Octo seems to work fine with this change, but I still keep getting spammed with this notification. I did the same investigation here to add `ssh_aliases`, but think there's something missing here.

**Reply by JohnWilliston** [reply]
> After some digging I managed to find the issue:
> 
> 1. I'm using ssh aliases in `~/.ssh/config` to link private keys to specific hosts.
> 2. I have an alias named `personal_github`, which points to `github.com`
> 3. `git remote` in this repository is configured as `git@personal_github:<username>/<repo>`
> 4. Octo grabs the `host_url` from the output of `git remote -v`, which returns `personal_github`
> 5. Then Octo runs `gh auth status --hostname personal_github`, which returns the message `You are not logged into any accounts on personal_github`.
> 
> The solution is to add the ssh alias to Octo's setup:
> 
> ```lua
>     require("octo").setup({
>         ssh_aliases = { personal_github = "github.com" },
>     })
> ```

Thank you so much for posting this! I've been trying to get Octo working for a while and was having the same issue!

**Reply by williambdean** [reply]
Great to hear @JohnWilliston 

Are you spammed with notifications like @PeterCardenas ?

**Reply by JohnWilliston** [reply]
Um, I wouldn't say I'm 'spammed' with notifications. I saw only the one FWIW.

**Reply by PeterCardenas** [reply]
this seems to be working for me now. I removed my notification silencing and no more spamming! not sure what happened but thanks for the consistent improvements 🙏
